src/Type/Php/StrSplitFunctionReturnTypeExtension.php

```diff
@@ -101,9 +101,6 @@ public function getTypeFromFunctionCall(FunctionReflection $functionReflection,
 		$items = isset($encoding)
 			? mb_str_split($stringValue, $splitLength, $encoding)
 			: str_split($stringValue, $splitLength);
-		if (!is_array($items)) {
```

Comment:
phpstan now knows $items will always be an non-empty-array. there this was rendered dead code with the changes of this PR.